vpn/options.go

```diff
@@ -390,14 +445,66 @@ func getOptionsFromLines(lines []string, dir string) (*Options, error) {
 		} else {
 			key, parts = p[0], p[1:]
 		}
-		e := parseOption(s, dir, key, parts)
+		e := parseOption(opt, dir, key, parts)
 		if e != nil {
 			return nil, e
 		}
 	}
-	return s, nil
+	return opt, nil
+}
+
+func isOpeningTag(key string) bool {
+	switch key {
+	case "<ca>", "<cert>", "<key>":
+		return true
+	default:
+		return false
+	}
+}
```

Comment:
Is the input file XML? If so, would it be possible to parse using `encoding/xml`?

Reply:
as discussed, it's not XML. the different blocks are identified by tags like:
```
<option>
---BEGIN BLOCK FOO---
...
---END BLOCK FOO
</option>
```
This is referred to as "inline file support" in openvpn's documentation. I will clarify the docs in minivpn about this.
I'm clarifying